test-content/themes/wp-empty-theme/style.css

```
@@ -0,0 +1,15 @@
+/*
+Theme Name: WP Empty Theme
+Theme URI: https://team.git.corp.google.com/dev-web-ecosystem/wp-empty-theme
+Author: Felix Arntz, Google
+Author URI: https://opensource.google.com
+Description: "Empty" WordPress theme for testing a barebones WordPress site, as a reference point to compare actual themes and plugins to.
+Requires at least: 5.3
+Tested up to: 5.8
```

```suggestion
Tested up to: 6.1.1
```

Not blocker, I'd like to hear @felixarntz's thoughts on this before the final review.

---

@mukeshpanchal27 I think we can potentially update that separately. Not a big deal really, given that this theme would not really be used and the end user wouldn't be able to see it.

I'd like to move ahead and get this PR merged, so we can consider doing that later. Nit-pick: I think we should stick to major WordPress releases in these annotations (i.e. make this `6.1` instead of `6.1.1`).